Import-VIRole.ps1

```diff
@@ -0,0 +1,114 @@
+﻿function Import-VIRole
+{
+    <#  
+            .SYNOPSIS
+            Imports a vSphere role based on pre-defined configuration values
+            .DESCRIPTION
+            The Import-VIRole cmdlet is used to parse through a list of pre-defined permissions to create a new role. Often, this is to support a particular vendor's set of requirements for access into vSphere.
```

Comment:
Can you add an .EXAMPLE and .INPUTS, similar to [this](https://github.com/rnelson0/vCenter-roles/blob/master/infrastructurenavigator.ps1#L6-L11)?